### ISSUE TITLE
v1.2.0 is broken

### ISSUE THREAD

**Issue by b1rdex** [Codeception/module-asserts#4]
```
Codeception PHP Testing Framework v4.1.4
 Powered by PHPUnit 8.5.3 by Sebastian Bergmann and contributors.
 Running with seed: 
 Fatal error: The modifiers of the trait method assertMatchesRegularExpression() are changed, but this method does not exist. Error in /builds/redacted/redacted/vendor/codeception/module-asserts/src/Codeception/Module/Asserts.php on line 11
```

**Comment by b1rdex** [comment]
Ok, the reason isn't related to PHPUnit version at all. The issue happened because I didn't update `lib-asserts` which introduced those methods.
This package still requires it as `^1.0` and the version constraint should be fixed to `^1.12`.

**Comment by Naktibalda** [comment]
Thanks, released as 1.2.1
